File: C++/Manual 02/Question04.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    int x;
    cout << "Enter the value of nth Term: ";
    cin >> x;

    if (x <= 0)
    {
        cout << "Invalid Input!";
    }
    else
    {
        int i = 1;
        double sum = 0;
        while (i <= x)
        {
            sum = sum + (1 / (pow(i, i)));
            cout << (1 / pow(i, i)) << endl;
            i++;
        }
        cout << "Sum of the series is: " << sum;
    }
}

File: JavaScript/Manual 02/Question04.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter n: ", function (n) {
    let sum = 0.0;

    for (let i = 1; i <= n; i++)
        sum += 1.0 / (Math.pow(i, i));

    console.log("The sum of the series is: " + sum);

    rl.close();
})